sqlite/sqlite.go

```diff
@@ -864,12 +867,19 @@ func (db *DB) ReplaceVoucher(ctx context.Context, guid protocol.GUID, ov *fdo.Vo
 	if err != nil {
 		return fmt.Errorf("error marshaling ownership voucher: %w", err)
 	}
+	if err := db.insert(ctx, "mfg_vouchers", map[string]any{
+		"guid":        ov.Header.Val.GUID[:],
+		"device_info": ov.Header.Val.DeviceInfo,
+		"cbor":        data,
+		"created_at":  time.Now().Unix(),
+		"updated_at":  time.Now().Unix(),
+	}, nil); err != nil {
+		return err
+	}
 	return db.update(ctx, "owner_vouchers",
 		map[string]any{
-			"guid":        ov.Header.Val.GUID[:],
-			"device_info": ov.Header.Val.DeviceInfo,
-			"cbor":        data,
-			"updated_at":  time.Now().Unix(),
+			"status":     int(fdo.VoucherStatusOnboardedNew),
+			"updated_at": time.Now().Unix(),
```

Comment:
This requires a transaction, but the SQLite implementation in go-fdo was designed to be transaction-less in order to support Cloudflare D1.